File: hw/cañarte/u3/HW28SingletonAndAbstractFactory/Java/GUIFactory/src/main/java/ec/edu/espe/guifactory/controller/WinButton.java
package ec.edu.espe.guifactory.controller;

/**
 *
 * @author Saray Canarte, Code Creators, DCCO-ESPE
 */
public class WinButton extends Button {
    @Override
    public void paint() {
        System.out.println("I'm a WinButton " + caption);
    }
}

File: hw/cañarte/u3/HW28SingletonAndAbstractFactory/Python/AbstractFactory/controller/WinButton.py
import sys
import os
# Saray Cañarte, codecreators, DCCO - ESPE

sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
from controller.Button import Button
class WinButton(Button):
    def paint(self):
        print(f"I'm a WinButton {self.caption}")
